net/gce.sh

```diff
@@ -95,6 +107,7 @@ Manage testnet instances
  config - configure the testnet and write a config file describing it
  delete - delete the testnet
  info   - display information about the currently configured testnet
+ status - display status information of all resources (colo-only)
```

Comment:
nit: remove `(colo-only)` here, and supply a stub implementation of `cloud_StatusAll` for the other three providers (it can error out)

Reply:
Sure thing!  I'm thinking we might actually want to spin this out to another `colo-admin.sh` script that handles some of the other orthogonal features the colo will require